utils.css

```diff
@@ -34,18 +54,50 @@
     position: absolute;
 }
 
+.mb-0 {
+    margin-bottom: 0;
+}
+
+.padding-02 {
```

Comment:
Let's refactor the paddings and have them in fixed increments: let's say `0.25em`. This way you can have `pt-1 = 0.25`, `pt-2 = 0.5` etc

After you do that, this rule should only be `px-8` meaning padding on the X axis, of 8 units = 2em